src/services/catalog.service.ts

```diff
@@ -158,6 +160,7 @@ export class ImageItem implements IServiceItem {
     this.longDescription = this.getLongDescription();
     this.catsBySubCats = this.getCategoriesBySubCategories();
     this.isBuilder = _.includes(this.tags, 'builder');
```

Comment:
https://github.com/openshift/origin-web-console/blob/master/app/scripts/services/catalog.js#L151-L160

The logic around all this is complicated unfortunately.